examples/run_selected_examples.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+APBS_EXE="apbs"
+BASE_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+export PATH="$BASE_PATH/../../../bin:$PATH"
+cd $BASE_PATH
+EXCLUDE="pygbe"
```

Comment:
@nsoblath we definitely don't want to exclude PYGBe in the actual release.

Reply:
That was a relic of the `run_examples.sh` script.  It's not actually used in `run_selected_examples.sh`.  Now cleaned up.